lib/generators/authentication/templates/controllers/api/identity/email_verifications_controller.rb.tt

```diff
@@ -5,6 +5,8 @@ class Identity::EmailVerificationsController < ApplicationController
 
   def edit
     @user.update! verified: true
+
+    head :no_content
```

Comment:
why this? is it really necessary?

Reply:
Yes - the test was failing without it. It appears it was expecting a view to be rendered otherwise:

```
Error:
Identity::EmailVerificationsControllerTest#test_should_verify_email:
ActionController::MissingExactTemplate: Identity::EmailVerificationsController#edit is missing a template for request formats: text/html
    test/controllers/identity/email_verifications_controller_test.rb:24:in `block in <class:EmailVerificationsControllerTest>'


rails test test/controllers/identity/email_verifications_controller_test.rb:21
```

Reply:
Oh, I found the problem, when generating the test app for api, uses `rails new test-app --api` and you don't need to use `rails generate authentication --api`

Reply:
Ok - let me update the test to reflect that

Reply:
Hey you know what? you were right, sorry 😂. can you revert the change ?

Reply:
Are you sure? It passed the CI run: 
https://github.com/kylekeesling/authentication-zero/actions/runs/3745808484/jobs/6360593579

Reply:
ok no problem, added back